pkgs/by-name/ni/nixos-render-docs-redirects/src/nixos_render_docs_redirects/__init__.py

```diff
@@ -0,0 +1,86 @@
+import json
+import argparse
+from pathlib import Path
+
+
+def add_new_identifier(redirects: dict[str, list[str]], identifier: str, path: str) -> dict[str, list[str]]:
+    if anchor in redirects:
+        print("The identifier in the given element already exists.")
+        exit(1)
+    redirects[anchor] = [args.element]
```

Comment:
Do we want to enforce some sorting here? Python dicts are internally ordered by insertion, which also translates to the JSON output unless overridden. We may or may not want to be slightly fancy here, and e.g. insert alphabetically. WDYT?

Reply:
Inserting alphabetically is what I tried first; and I got a diff as big as the file itself. Neither redirect JSONs are sorted by alphabet, I think it's in the order identifiers were found in.

I'm open to enforcing some sorting here, but alphabetic *seems* off the table (unless we want to merge a separate PR on sorting it first -- causing practically unresolvable merge conflicts on all PRs that update the JSON).
If not alphabetical, then what type of sorting can we do?

Reply:
Being fancy could for instance mean inserting alphabetically at the first instance, i.e. start from empty dict, `for key, value in iterate(redirects)` and insert when `identifier > key`, then iterating over the rest of the keys and insert those.

Otherwise we can of course append-only, and just define this file to only be read by humans to take note of the diff (most checks should be automatic anyway).

Reply:
Opted for the fancy route